lib/tbot/config/destination_directory.go

```diff
@@ -121,11 +123,19 @@ func mkdir(p string) error {
 	stat, err := os.Stat(p)
 	if trace.IsNotFound(err) {
 		if err := os.MkdirAll(p, botfs.DefaultDirMode); err != nil {
+			//If the user running does not have permission to create the bot dir
+			if errors.Is(err, fs.ErrPermission) {
+				return trace.Wrap(err, "Teleport does not have permission to write to: %v. Ensure that you are running as a user with appropriate permissions.", p)
+			}
 			return trace.Wrap(err)
 		}
 
 		log.Infof("Created directory %q", p)
 	} else if err != nil {
+		//This error permission can occur if unable to read into the data dir
+		if errors.Is(err, fs.ErrPermission) {
+			return trace.Wrap(err, "Teleport does not have permission to write to: %v. Ensure that you are running as a user with appropriate permissions.", p)
```

Comment:
In this case, we weren't trying to write, so the error message is not correct.

I would say something like `does not have permission to access %v`.

Reply:
@zmb3 changed error message and applied other suggestions